components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.scss

```diff
@@ -38,6 +38,10 @@ chef-alert {
   justify-content: left;
 }
 
+chef-option.filter.unknown {
+  margin-right: 50px; // forces remaining filters into new row
+}
+
```

Comment:
per the new design spec for the connected filter option, this is a simple way to force the new row to happen, placing connected and disconnected on their on row.

Reply:
The other option is to put in an extra div with 0 height and 100% width which effectively works like ye olde `<br>`, is that true?

Reply:
My concern here is that you are manipulating the line assuming a constant panel width. If that panel is changed in the future the desired behavior here might be lost. So I would also lean toward something that forces an actual line break.

Reply:
@danielsdeleo - you absolutely can 👍 
@msorens - I see your point, my goal was to add less code, and given that there are no current plans to change anything, simple CSS could give us what we needed.

I'm going to change this to Dan's method above and we'll be all set 👍